src/test/java/com/games/toki/pot/domain/mapper/BusinessFlightToFlightModelConverterTest.java

```diff
@@ -0,0 +1,29 @@
+package com.games.toki.pot.domain.mapper;
+
+import com.games.toki.pot.domain.model.Flight;
+import com.games.toki.pot.service.response.BusinessFlight;
+import org.junit.Test;
+import reactor.core.publisher.Flux;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by mmundra on 1/17/19.
+ */
```

Comment:
in some of files there is this information. It's not a bug, but it's important to keep one approach. Or keep this in all files, or nowhere